mutex.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// gcslock is a scalable, distributed mutex that can be used to 
```

Comment:
It's a good practice to run [golint](https://github.com/golang/lint) on your source code to fish those out :)